.gitignore

```diff
@@ -1,11 +1,3 @@
-# buildout
```

Comment:
Isn't buildout still being used?

Reply:
I noticed this as well, but this gitignore is a straight copy from our [template](https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.gitignore), which does not contain the buildout stuff. I assumed it was intentional.